pkg/core/blockchain.go

```diff
@@ -3331,17 +3331,20 @@ func (bc *Blockchain) verifyHeaderWitnesses(currHeader, prevHeader *block.Header
 
 // GoverningTokenHash returns the governing token (NEO) native contract hash.
 func (bc *Blockchain) GoverningTokenHash() util.Uint160 {
-	return bc.contracts.NEO.Hash
+	// TODO: such replacement is valid iff we're not going to change their names and other hashable fields.
```

Comment:
Looks like we have a number of ties that hold us from _complete_ contract replacement. At the same time our primary targets are NEO/GAS and _additional_ contracts. In that sense all of the oracle/designation and other dances like `UtilityTokenHash()` are mostly OK.